### ISSUE TITLE
Update GeoIP database?

### ISSUE THREAD

**Issue by gnikandrov** [chubin/wttr.in#135]
Hello,

It would be nice to update GeoIP database on server from time to time.
Fresh GeoLite2 database correctly shows my IP address (in 89.179.112.0/20) as St. Petersburg, whereas your service returns data for Moscow.

Anyway, thanks for the great work.

**Comment by chubin** [comment]
Yes, it's a good idea. On the other hand, the majority of IP addresses are not in the database anyway, and for those addresses, we use an external service (ip2location) for geolocating (but your address was found in the local GeoLiite2  database, I think).

Thank you for reporting!

**Comment by gnikandrov** [comment]
Oh, I didn't bother to check ip2location. Yes, it returns obsolete data (Moscow). Anyway, fresh GeoLite2-City should solve the problem.

Thanks again.

**Comment by gnikandrov** [comment]
Ooops. Now it says that I'm from Port Credit, Canada :)
The IP is still the same, and recent GeoLite databases seem to be correct.

**Comment by chubin** [comment]
Could you please test it once again?

**Comment by gnikandrov** [comment]
@chubin Oh, now it works just fine! Thank you!
